tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = py26,py27,lint,docs
+skipsdist = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}/readthedocs:{toxinidir}
+    DJANGO_SETTINGS_MODULE=settings.test
+deps = -r{toxinidir}/requirements/pip.txt
+changedir = {toxinidir}/readthedocs
+commands =
+    py.test {posargs}
+
+[testenv:docs]
+changedir = {toxinidir}/docs
+commands =
+    sphinx-build -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html
```

Comment:
Yes, indeed. I dropped the option as we have some warnings now, I figured this could be addressed with another PR later